server/dataone_package.py

```diff
@@ -192,12 +216,13 @@ def add_object_record(name, description, size, object_format):
         add_object_record(item['name'], item['description'], item['size'], object_format)
 
     # Add a section for the tale.yml file
-    if file_sizes.get('tale_yaml') is not int() or None:
+    if file_sizes.get('tale_yaml') != int() or None:
```

Comment:
I'd use `not isinstance(file_sizes.get('tale_yaml'), int)`

Reply:
Ah since get can return None, it can check both at once. Nice.